plugins/MantisCoreFormatting/MantisCoreFormatting.php

```diff
@@ -190,7 +190,7 @@ function formatted( $p_event, $p_string, $p_multiline = true ) {
 		$t_string = mention_format_text( $t_string, /* html */ true );
 
 		# Process Markdown
-		if( ON == $s_markdown && $p_multiline ) {
+		if( ON == $s_markdown ) {
```

Comment:
This introduces side effects, e.g. check rendering of summary field on "My View" page. or "View Issue Details".
There are added new lines.